src/test/regress/expected/ref_citus_local_fkeys.out

```diff
@@ -79,12 +76,9 @@ NOTICE:  executing the command locally: SELECT l1 FROM ref_citus_local_fkeys.cit
 -- drop constraint for next commands
 ALTER TABLE citus_local_table DROP CONSTRAINT fkey_local_to_ref;
 NOTICE:  executing the command locally: SELECT worker_apply_inter_shard_ddl_command (1506002, 'ref_citus_local_fkeys', 1506001, 'ref_citus_local_fkeys', 'ALTER TABLE citus_local_table DROP CONSTRAINT fkey_local_to_ref;')
-NOTICE:  creating a new table for ref_citus_local_fkeys.citus_local_table
-NOTICE:  Moving the data of ref_citus_local_fkeys.citus_local_table
+NOTICE:  Removing table ref_citus_local_fkeys.citus_local_table from metadata as it is not connected to any reference tables via foreign key
```

Comment:
foreign key -> foreign keys

maybe also: Removing -> removing per the usual convention

Reply:
will address those before merging